src/utils/data_loaders.py

```diff
@@ -153,6 +153,103 @@ def targs_to_idx(col_name):
             'ix_to_knowledge_dic': ix_to_knowledge_dic
             }
 
+class TSVLoader(object):
```

Comment:
How does this relate to what Yada pushed recently?

Reply:
Previously, tsv loader didn't use pandas. I modified her pandas version.

Reply:
I'm not sure I understand. We really shouldn't have two TSV readers unless they're doing very different things. How is this different from what's currently in master?

Reply:
Does this looks better now? I removed the TSVLoader object, put its functionality into load_tsv.